src/recensio/plone/profiles/default/registry/settings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<registry>
+  <records interface="recensio.plone.interfaces.IRecensioSettings" />
```

Comment:
please add a prefix like `prefix="recensio.plone.settings"`

Reply:
did it and I think it's a good idea giving us clean namespaces. however, settings now need always the prefix `recensio.plone.settings`.

Reply:
Sorry, can you explain?

Reply:
in the old code we do not have this prefix. now we need to add it whereever we use a setting. You can see this in this commit: https://github.com/syslabcom/recensio.plone/pull/62/commits/b8843a767d39340df00e5e2085221f27a649e5b5

but it's not a big deal, I'm fine with it.

Reply:
Uhm, I see your point and I am thinking about removing the prefix now...